client/my-sites/upgrades/checkout-thank-you/google-voucher/terms-and-conditions.jsx

```diff
@@ -12,7 +17,12 @@ const googleTermsAndConditions = () => {
 			<ol>
 				<li className="google-voucher__terms-and-conditions">{ i18n.translate( 'Offer available to customers with a billing address in the United States only. One promotional code per advertiser.' ) }</li>
 
-				<li className="google-voucher__terms-and-conditions">{ i18n.translate( 'To activate this offer: Enter the promotional code in your account before December 31 2016. In order to participate in this offer, you must enter the code within 14 days of your first ad impression being served from your first AdWords account.' ) }</li>
+				<li className="google-voucher__terms-and-conditions">{
+					i18n.translate(
+						'To activate this offer: Enter the promotional code in your account before %(expirationDate)s. In order to participate in this offer, you must enter the code within 14 days of your first ad impression being served from your first AdWords account.',
```

Comment:
Alternate string suggestion: 
* `translate( 'To activate this offer: Enter the promotional code in your account before December 31 2016. In order to participate in this offer, you must enter the code within 14 days of your first ad impression being served from your first AdWords account.' )`&mdash; translations: **17**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/update-google-adwords-texts)